scripts/export.js

```diff
@@ -44,7 +44,14 @@ $(document).ready(() => {
         if (document.getElementById('feature-check').checked) {
             options.push('feature=' + $("#feature-select option:selected")[0].value);
         }
-        window.location.href = '/export?' + options.join('&');
+        let exportRef = '/export?' + options.join('&');
+        $.get(exportRef, downloadFile => {
```

Comment:
I totally agree, but I'm really not sure how we can add error handling otherwise. Definitely something to investigate.